ocean/nemo/nemolite2d/psykal/ocl_transform.py

```diff
@@ -5,24 +5,17 @@ def trans(psy):
 
     from psyclone.psyGen import TransInfo
     tinfo = TransInfo()
+    globaltrans = tinfo.get_trans_name('KernelGlobalsToArguments')
     cltrans = tinfo.get_trans_name('OCLTrans')
-    verbose = False
 
-    # First invoke
+    # Get the invoke
     schedule = psy.invokes.get('invoke_0').schedule
-    cltrans.apply(schedule)
-
-    # Map of OpenCL options
-    ocl_options =  {'local_size': 4}
 
-    # Attach options object to Kernel Call and Kernel Schedule
+    # Remove the globals from inside each kernel
```

Comment:
Is there a reason for removing the setting of 'local_size'?

Reply:
I couldn't see any performance improvement from it, so for the baseline version I prefer to go with the defaults. I plan to write an optimized psyclone script in the future, but first I need to find which parameters/values provide a real benefit.

Reply:
Makes sense.